contracts/Vault.vy

```diff
@@ -581,6 +581,13 @@ def setWithdrawalQueue(queue: address[MAXIMUM_STRATEGIES]):
         if queue[i] == ZERO_ADDRESS and self.withdrawalQueue[i] == ZERO_ADDRESS:
             break
         assert self.strategies[queue[i]].activation > 0
+
+        for j in range(MAXIMUM_STRATEGIES):
+            if queue[j] == ZERO_ADDRESS and self.withdrawalQueue[j] == ZERO_ADDRESS:
+                break
+            if queue[i] == queue[j] and i != j:
+                raise "duplicates in queue"
```

Comment:
Would be cheaper to do this outside of the loop

Reply:
You mean having a second `for i loop` and inside a `jor j loop`?

Reply:
Yeah but removing the `self.withdrawalQueue[j]` line, which adds `2000 * n^2` gas, where `n` is the length of the queue.

Reply:
self.withdrawalQueue[j] == ZERO_ADDRESS is a mistake I have removed it.